docs/blog/contributor-development-maintenance-guide.en-US.md

```diff
@@ -0,0 +1,155 @@
+
+4. **Compatibility between old and new**: Now that we're done with the early tips, it's important to make sure that the new property and the old one live in harmony for a while until the old one dies. That is to say, currently we use expired property to work normally, in addition to the warning, can not have any changes.
+
+   ![007](https://user-images.githubusercontent.com/10286961/224358439-76c42c78-e244-42bd-8935-b08f536931a2.png)
+
+   We can do something like this, making sure that the new property takes precedence over the old one, and only trying to use the old one if no new one exists.At this point, our property expiration work is done.
+
+5. **Wipe out**: After our properties have expired for a certain period of time, usually when the major version number is upgraded, we can remove the obsolescence. We also need to delete comments, warnings, test cases, and documents that were added to obsolete the property etc. At this point our property abandonment work is done.
+
+## Take it to the next level
+
+After contributing for some time, I believe you have gained a deeper understanding of Ant Design's overall development process and project architecture. At this point, you may want to take on more tasks and contribute more to the open source community.Then, you can apply to be a Collaborator at Ant Design by commenting on the link below for more active contributors. [Add Collaborator permission for some active contributors](https://github.com/ant-design/ant-design/issues/3222), The Collaborators will then start the voting process, and after passing the voting, they will invite you to officially become a Collaborator at Ant Design.
+
+After you become a newly appointed Ant Design collaborator, you will be assigned some additional rights, such as:
+
+- Tag the issue
+- Close the issue
+- Free to create branches under the Ant Design project
+- Free to merge approved PR
+
+- ...
+
+There are some caveats to this.
+
+### Merge PR
+
+At this point, you can merge the approved PR, but merging code is tricky, knowing when to use squash and when not to.
+
+![008](https://user-images.githubusercontent.com/10286961/224358476-2332e36f-0adf-486f-8b17-1b2ad34926aa.jpg)
+
+- **Merge pull request**: Normally merge the current PR into the target branch usually when branch to branch merge each other, do not merge the commit record, do not merge PR, or you will forget to add the target branch too many redundant commit records.
+- **Squash and merge**: Merge commit records into a squash and merge into the target branch(this is usually used for merging PR).
+- **Rebase and merge**：When you want to adjust the baseline
+
+Before merging branches, it is a good idea to check with the author that they have been fully modified and that they have been carefully reviewed. And at least one maintainer approves this PR. Determine which of the above scenarios this PR is, and then choose the appropriate way to merge.
```

Comment:
Merge in Ant Design requires at least one collaborator's approve. Collaborator who approved should ensure PR are fully reviewed and confirm.